IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectiveness.mo

```diff
@@ -83,7 +83,7 @@ equation
            fra_a2 * Medium2.specificHeatCapacityCp(state_a2_inflow) +
            fra_b2 * Medium2.specificHeatCapacityCp(state_b2_inflow) else
         Medium2.specificHeatCapacityCp(state_a2_inflow));
-  CMin_flow = min(C1_flow, C2_flow);
+  CMin_flow = IBPSA.Utilities.Math.Functions.smoothMin(x1=C1_flow, x2=C2_flow, deltaX=CMin_flow_small);
```

Comment:
A better solution could then be to use a cubic/quintic Hermite spline? Since these should not have an overshoot or undershoot?